controlcenter/templates/controlcenter/widgets/itemlist.html

```diff
@@ -1,6 +1,6 @@
 {% load controlcenter_tags %}
 <table class="controlcenter__table" {% if widget.sortable and widget.values|length > 1 %}data-sortable{% endif %}>
-    {% if widget.sortable and widget.list_display and widget.values %}
+    {% if widget.list_display and widget.values %}
```

Comment:
but the header is shown if there are columns defined in `list_display` and there are `values`. sortable has nothing to do with this.

Reply:
Wow. Thanks!